android-r.md

```diff
@@ -0,0 +1,95 @@
+
+### Does it concern me?
+
+If you're using or want to update to the **SDK 41** or **higher**, you should check if you're using location or media library and follow the instruction below.
+
+You may also observe that some functionalities in `Expo Go` don't work as intended when using a device with Android R or higher. That's because we didn't add changes required by the newest Android to the older SDKs. Therefore, when you're building your app using older SKD via our build service, everything should work as it used to.
+We didn't want to break existing applications and force developers to change their code even when they didn't update SDK.
+
+## A new way of handling location permissions
+
+Android team to protect user privacy changed how the app is getting access to location in the background. To address those changes in the Expo SDK, we split location permissions into `foreground` and `background`.
+
+### The foreground permission
+
+If your app contains a feature that shares or receives location information only once, or for a defined amount of time, then that feature requires foreground location access.
+
+#### Requesting permission
+
+When your application requests the foreground permission, users will see modal that looks like this:
+
+[<img src="./assets/android-r/foreground_permissions_modal.png" width="300" />](./assets/asc-app-id/finding-app-id.png)
+
+#### Differences in the way it works across all Android versions
+
+The flow for that permission is the same across all android versions.
+
+### The background permission
+
+An app requires background location access if a feature within the app constantly shares location with other users or uses the Geofencing API.
+
+#### Requesting permission
+
+When your application requests the foreground permission, users won't see modal. He will be redirect to the settings screen which should look like this:
+
+[<img src="./assets/android-r/background_permissions_modal.png" width="300" />](./assets/asc-app-id/finding-app-id.png)
+
+That's why before that, you should inform the user why your application requires background permissions.
+
+**You can request the background permission only if the foreground permission was granted.**
+
+#### Differences in the way it works across all Android versions
+
+- On Android lower than Q, the background permission resolves to the foreground permission and the application won't redirect the user to the settings screen when asked.
+
+- On Android Q, the background permission will show up a similar system modal to the foreground permissions instead of redirecting to the settings screen.
+
+> If you want to know more about location permissions on Android, please check out this [article](https://medium.com/@ty2/understanding-permissions-for-background-location-on-android-11-and-below-bc3ad9be320a).
+
+## Scoped Storage
+
+It’s a concept of storing files, images, etc. separately, called Collections, which restrict the conventional access of the whole storage.
```

Comment:
```suggestion
Scoped storage is another change that was made in the interest of protecting Android users. Rather than granting an app access to all public files on the device filesystem when it needs to write files, it can be given access to an app-specific scoped directory and pre-defined locations for files, images, etc, called Collections.
```

does this wording make sense?